x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/edit_job_flyout/edit_job_flyout.js

```diff
@@ -73,10 +77,37 @@ class EditJobFlyoutUI extends Component {
     }
   }
 
-  closeFlyout = () => {
+  closeFlyout = (isConfirmed = false) => {
```

Comment:
Is it possible to listen for the user closing the flyout by the cross icon top right? Clicking the cross icon closes the flyout but doesn't seem to go via this `closeFlyout` function.

Reply:
Fixed in https://github.com/elastic/kibana/pull/47926/commits/5309d376ad9395a0d4b19881e4ff7f90e12d7a62